lib/oli/publishing/delivery_resolver.ex

```diff
@@ -135,6 +134,22 @@ defmodule Oli.Publishing.DeliveryResolver do
     |> emit([:oli, :resolvers, :delivery], :duration)
   end
 
+  def pages_with_attached_objectives(section_slug) do
+    page_id = Oli.Resources.ResourceType.get_id_by_type("page")
+
+    fn ->
+      from([s: s, sr: sr, rev: rev] in section_resource_revisions(section_slug),
+        where:
+          rev.resource_type_id == ^page_id and
+            fragment("? != '[]'", rev.objectives["attached"]),
+        select: rev
```

Comment:
Instead of selecting the entire revision, can you change this to only select the page resource_id and the objectives (the only two things that the callers of this actually need).  Currrently this selects the entire revision, which contains the potentially large `content` field. I was hoping to avoid selecting that content field.